<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>find_color_block</class>
 <widget class="QWidget" name="find_color_block">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1306</width>
    <height>934</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_16">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label_img">
       <property name="minimumSize">
        <size>
         <width>640</width>
         <height>480</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
border-image:url(./img_icon/1.jpg);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="buddy">
        <cstring>label_img</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_img_gray">
       <property name="minimumSize">
        <size>
         <width>640</width>
         <height>480</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
border-image:url(./img_icon/1.jpg);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="minimumSize">
        <size>
         <width>640</width>
         <height>200</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>3ds</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>DexArm</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_5">
            <property name="font">
             <font>
              <family>3ds</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="title">
             <string>Adjustment</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="font">
                  <font>
                   <family>3ds</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>step unit</string>
                 </property>
                 <property name="buddy">
                  <cstring>spinBox</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QPushButton" name="Button_y_add">
                 <property name="text">
                  <string>Y+</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QPushButton" name="Button_x_add">
                   <property name="text">
                    <string>X+</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Button_z_add">
                   <property name="text">
                    <string>Z+</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Button_z_less">
                   <property name="text">
                    <string>Z-</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Button_x_less">
                   <property name="text">
                    <string>X-</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="Button_y_less">
                 <property name="text">
                  <string>Y-</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_12">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QPushButton" name="Button_rest">
                   <property name="text">
                    <string>REST</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Button_home">
                   <property name="text">
                    <string>HOME</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="Button_arm_start">
                 <property name="text">
                  <string>Start</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_10">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="1">
               <widget class="QTextBrowser" name="show_red_place">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_9">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Blue place</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QTextBrowser" name="show_blue_place">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_10">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Yellow place</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QTextBrowser" name="show_yellow_place">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_8">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Red place</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>108</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_6">
              <property name="font">
               <font>
                <family>3ds</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="title">
               <string>Color Block Demo</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_13">
                 <item>
                  <widget class="QComboBox" name="Boxdemo"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_11">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_7">
              <property name="title">
               <string>XY Multiple</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_15">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_14">
                 <item>
                  <layout class="QFormLayout" name="formLayout_4">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_12">
                     <property name="font">
                      <font>
                       <family>3ds</family>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>x multiple</string>
                     </property>
                     <property name="buddy">
                      <cstring>spinBox</cstring>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QSpinBox" name="Box_x"/>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_13">
                     <property name="font">
                      <font>
                       <family>3ds</family>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>y multiple</string>
                     </property>
                     <property name="buddy">
                      <cstring>spinBox</cstring>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QSpinBox" name="Box_y"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Button_xy_ok">
                   <property name="text">
                    <string>ok</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>150</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>3ds</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Color choose</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QCheckBox" name="checkBox_Red">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>red</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_Blue">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>blue</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_Yellow">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>yellow</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="font">
            <font>
             <family>3ds</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Result</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QTextBrowser" name="show_recong_result">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>80</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <family>3ds</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="title">
          <string>HSV Adjustment</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>H Max</string>
                </property>
                <property name="buddy">
                 <cstring>H_Slider_max</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="H_Slider_max">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>H Min</string>
                </property>
                <property name="buddy">
                 <cstring>H_Slider_min</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="H_Slider_min">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>S Max</string>
                </property>
                <property name="buddy">
                 <cstring>S_Slider_max</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="S_Slider_max">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>S Min</string>
                </property>
                <property name="buddy">
                 <cstring>S_Slider_min</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="S_Slider_min">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>V Max</string>
                </property>
                <property name="buddy">
                 <cstring>V_Slider_max</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="V_Slider_max">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>V Min</string>
                </property>
                <property name="buddy">
                 <cstring>V_Slider_min</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="V_Slider_min">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>HSV Range</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="TextEdit_hsv">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>80</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="plainText">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
